app.js

```diff
@@ -114,19 +105,24 @@ function submit()
 {
   try
   {
-    var deck_name    = document.getElementById("deck_name").value;
+    var deck_name = document.getElementById("deck_name").value;
```

Comment:
Not a fan of alignment? :D

Reply:
I'm a bigger fan of one space around operators, but your project, your code style,I guess.

Reply:
This code was written in a couple of hours and is very short, I don't particularly care about code style here, especially since I don't write JavaScript. :rdwut: